File: benchmark/fib.js
function fib(n) {
  if (n <= 1) return n;
  return fib(n - 1) + fib(n - 2);
}

const start = performance.now();
const result = fib(35);
const end = performance.now();

console.log(`RESULT=${result}`);
console.log(`TIME_MS=${Math.round(end - start)}`);

File: benchmark/fib.lua
local function fib(n)
  if n <= 1 then
    return n
  end
  return fib(n - 1) + fib(n - 2)
end

local start = os.clock()
local result = fib(35)
local end_time = os.clock()

local elapsed_ms = math.floor((end_time - start) * 1000)
print("RESULT=" .. result)
print("TIME_MS=" .. elapsed_ms)
